src/main/webapp/app/exercises/shared/structured-grading-criterion/grading-instructions-details/grading-instructions-details.component.html

```diff
@@ -1,40 +1,108 @@
+                        <div class="input-group" style="height: 33px; width: 75%">
+                            <input class="form-control" type="text" [(ngModel)]="criteria.title!" (change)="onCriteriaTitleChange($event, criteria)" placeholder="Criteria Title" />
+                            <span class="re-evaluate-button" id="reset-criteria-button" (click)="resetCriteriaTitle(criteria)">
+                                <fa-icon [icon]="'undo'"></fa-icon>
+                            </span>
+                            <div class="col-12 col-sm-6 col-xl-3 mb-1">
+                                <span class="btn btn-block btn-danger" id="delete-criteria-button" (click)="deleteGradingCriteria(criteria)">
+                                    <fa-icon [icon]="'trash'"></fa-icon>
+                                    <span jhiTranslate="artemisApp.exercise.deleteGradingCriteria"></span>
+                                </span>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+                <div *ngIf="criteria.structuredGradingInstructions!">
+                    <div *ngFor="let instruction of criteria.structuredGradingInstructions">
+                        <div class="instructions">
+                            <jhi-markdown-editor
+                                #markdownEditor
+                                [markdown]=""
+                                [metaCommands]="[]"
+                                [defaultCommands]="[]"
+                                [headerCommands]="[]"
+                                [colorCommands]="[]"
+                                [showEditButton]="false"
+                                [showPreviewButton]="false"
+                                [domainCommands]="domainCommandsGradingInstructions"
+                                (markdownChange)="prepareForSave()"
+                                (textWithDomainCommandsFound)="onInstructionChange($event, instruction)"
+                                class="editor"
+                                style="width: 65%"
+                            ></jhi-markdown-editor>
+                            <div class="form-group text-center" style="width: 25%">
+                                <span class="re-evaluate-button" id="reset-button" (click)="resetInstruction(instruction, criteria)">
+                                    <fa-icon [icon]="'undo'" size="2x"></fa-icon>
+                                </span>
+                                <span class="re-evaluate-button" id="delete-button" (click)="deleteInstruction(instruction, criteria)">
+                                    <fa-icon [icon]="'trash'" size="2x"></fa-icon>
+                                </span>
+                                <span class="re-evaluate-button" id="add-button" (click)="addNewInstruction(criteria)">
+                                    <fa-icon [icon]="'plus'" size="2x"></fa-icon>
```

Comment:
Do we ever use icons of size 2x? I would stay with the 1x size, but that is personal taste I guess :)

Reply:
👍 Personally, I like the minimal design. I mostly copied the design from the re-evaluation of quizzes (added screenshot). I just thought that if this is approved for quiz re-evaluation, I can use it for my work. If you think that small size fits better, i can change it quickly.

<img src="https://user-images.githubusercontent.com/23650630/117786297-a2dfde00-b245-11eb-80fb-3abd2ecd053b.png" width="400"></img>